js/ui/scheduler/utils.js

```diff
@@ -52,14 +52,22 @@ const isTimezoneChangeInDate = (date) => {
     return (startDayDate.getTimezoneOffset() - endDayDate.getTimezoneOffset()) !== 0;
 };
 
+const isDifferentDates = (startDate, endDate) => {
```

Comment:
Looks like we extend "dateUtils.sameDate". In this case, the method should be fixed to "isSameDate".

Reply:
Everywhere in the code we call a dateUtils.sameDate. So, I gave this name (isDifferentDates ) to reduce confusion.

Reply:
isSameAppointmentDates(startDate, endDate) ?